src/components/PostPageDetail/index.js

```diff
@@ -67,9 +66,9 @@ const PostPageDetailIdComponent = (props) => {
   const [statusUpvote, setStatusUpvote] = React.useState(false);
   const [statusDownvote, setStatusDowvote] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [item, setItem] = React.useState(null);
   const navigation = useNavigation();
   const route = useRoute();
+  const [item, setItem] = React.useState(route?.params?.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Initializing the `item` state with `route.params.data` is a good practice as it avoids unnecessary re-renders caused by setting the state later in the component lifecycle.
```diff
- const [item, setItem] = React.useState(null);
+ const [item, setItem] = React.useState(route?.params?.data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->